tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -282,13 +268,13 @@ public function theUserReloadsTheUsersPage() {
 	}
 
 	/**
-	 * @When the administrator/user disables the user :username using the webUI
+	 * @When the administrator/user disables user :username using the webUI
```

Comment:
This is the change that makes a real difference - ``disableUsers.feature`` has skipped steps that were ported from core ``stable10``. They need this change so that they can pass.